qemu/tests/openflow_test.py

```diff
@@ -29,10 +29,10 @@ def run_tcpdump_bg(session, addresses, dump_protocol):
         """
         tcpdump_cmd = "killall -9 tcpdump; "
         tcpdump_cmd += "tcpdump -iany -n -v %s and 'src %s and dst %s' -c 1 &"
-        session.cmd_output_safe(tcpdump_cmd % (dump_protocol,
-                                               addresses[0], addresses[1]))
+        session.sendline(tcpdump_cmd % (dump_protocol, addresses[0],
+                                        addresses[1]))
```

Comment:
If you use `sendline` instead of `cmd_output_safe`, I will recommend to use a separated session for it, cause you will have potential risk that the command does return. And at somewhere you does not need the process(tcpdump) anymore, try killing the process if it is still there and close the session would be safer.